src/routes/login/+page.svelte

```diff
@@ -84,31 +139,57 @@
                   title="Klicke, um einen neuen Account anzulegen">
                 Noch keinen Account?
             </Link>
+            <Link href="#"
+                  title="Klicke, um dein Passwort zurück zu setzen."
+                  on:click={() => {changeState(State.ResetPasswordPending)}}
+            >
+                Passwort vergessen?
+            </Link>
             <Button classes="login-button"
                     type="submit"
                     ariaLabel="Klicke zum Anmelden">Anmelden
             </Button>
         </form>
-    </div>
-
-    <Footer globals={data.globals}
-            menu={Menu.footerOut} />
-</div>
+    {:else if state === State.ResetPasswordPending}
+        <form class="login-form-width-wrapper"
+              on:submit|preventDefault={resetPassword}>
+            <HeadlinePage>Passwort zurücksetzen</HeadlinePage>
+            <ErrorMessage message={errorMessage} />
+            <Input
+                  classes="login-username-mail login-username-mail-reset-extra"
+                  id="login-username-or-email"
+                  type="text"
+                  labelText="Nutzername oder E-Mail:"
+                  placeholderText="Nutzername oder E-Mail"
+                  ariaLabel="Gib den Nutzernamen oder die E-Mail ein"
+                  bind:value={usernameOrEmail}
+            />
+            <Button classes="login-button"
+                    type="submit"
+                    ariaLabel="Klicke, um dein Passwort zurückzusetzen">Passwort zurücksetzen
+            </Button>
+            <Link href="#"
+                  title="Klicke, wieder zum login zu kommen."
+                  on:click={() => {changeState(State.Login)}}
+            >
+                zurück
+            </Link>
+        </form>
+    {:else if state === State.ResetPasswordSuccess}
+        <form class="login-form-width-wrapper login-form-gap-wrapper"
+              on:submit|preventDefault={() => {changeState(State.Login);}}>
+            <HeadlinePage>Passwort zurückgesetzt</HeadlinePage>
+            <Paragraph classes="paragraph-pre-wrap"
+                       --text-align="center">{'Dein Passwort wurde erfolgreich zurück gesetzt.\nDu erhältst eine Mail mit einem Link.\nDort kannst du dein neues Passwort eingeben.\nDanach kannst du dich mit deinem neuen Passwort wieder anmelden.'}</Paragraph>
+            <Button classes="login-button"
+                    type="submit"
+                    ariaLabel="Klicke, um dich anzumelden">Login
```

Comment:
On the other buttons, the text is "Anmelden" instead of "Login".
```suggestion
                    ariaLabel="Klicke, um dich anzumelden">Anmelden
```